pkgs/by-name/ze/zeekstd/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  fetchFromGitHub,
+  lib,
+  rustPlatform,
+}:
+let
+  version = "0.2.2";
+in
+rustPlatform.buildRustPackage {
+  pname = "zeekstd";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "rorosen";
+    repo = "zeekstd";
+    rev = "v${version}";
+    hash = "sha256-Blyp5GpnytB3S4k6lp2fAwXueaUtXqPW+WLEmFNPZc0=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-bbl0zHxd2HYkctX029mtxDciC2tnPVTlHxYyetmtuw0=";
+  stripAllList = [ "bin" ];
+
+  meta = with lib; {
+    description = "A CLI tool that works with the zstd seekable format";
+    homepage = "https://github.com/rorosen/zeekstd";
```

Comment:
add changelog

Reply:
Thanks for the review, I added the changes